resources/views/components/game/claim-info.blade.php

```diff
@@ -76,20 +70,20 @@
             $activity = 'played this game';
         } else {
             // player_sessions only exist after 14 Oct 2023
-            $achievement = $achievementSetClaim->game->achievements
+            $achievement = Achievement::where('GameID', $achievementSetClaim->game->id)
```

Comment:
I'm pretty sure we can remove this code, a claim had to be renewed every three months, and Oct 2023 is like 15 months ago. Wouldn't recommend doing that in this PR though.

Reply:
Yea, the only way I could personally fall into this branch is if I started doing stuff without ever playing the game (like editing a code note on the website). I was wondering what the actual utility of it was now that we have player sessions. Probably best to just get this logic when we finally get to the game page React migration.